tests/unit/composables/Project.spec.js

```diff
@@ -1,15 +1,60 @@
-import { createProject } from 'src/composables/Project';
+import {
+  createProjectTemplate,
+  getProjects,
+  getProjectById,
+  saveProject,
+  deleteProjectById,
+  deleteAllProjects,
+} from 'src/composables/Project';
```

Comment:
Each test should have is own unit test(s) (and without calling another of thoose functions)